src/main/resources/db/migration/V4_8__create_table_mote_referat.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE MOTE_REFERAT (
+	id                       SERIAL             PRIMARY KEY,
+	uuid                     VARCHAR(50)        NOT NULL UNIQUE,
+	created_at               TIMESTAMP          NOT NULL,
+	updated_at               TIMESTAMP          NOT NULL,
+	mote_id                  INTEGER REFERENCES MOTE (id) ON DELETE CASCADE,
+	arbeidstaker_stilling    TEXT,
+	situasjon                TEXT,
+	konklusjon               TEXT,
+	arbeidstaker_oppgave     TEXT,
+	arbeidsgiver_oppgave     TEXT,
+	veileder_oppgave         TEXT,
+	document                 JSONB NOT NULL DEFAULT '[]'::jsonb
+);
+
+CREATE TABLE MOTE_EKSTRA_DELTAKER (
```

Comment:
Hva er en ekstra deltaker? Støttetvi ikke bare arbeidstaker og arbeidsgiver nå?

Reply:
I Figma-skissene er det lagt til noe for å kunne registrere andre som har deltatt i møtet - type Verneombud etc.

Reply:
@janzh Kanskje motedeltaker_annen er et bedre navn på denne tabellen?